source/_integrations/apsystems.markdown

```diff
@@ -26,6 +27,7 @@ The following data is provided by the integration:
 The following data can be set by the integration:
 
 - Maximal output in watts
+- Inverter status (on or off)
```

Comment:
I realize this comment goes beyond the scope of this PR. But I noticed there are quite a few sensors now that are not described in the docs. I think it could be useful to have separate sections for sensors, numbers, and switch and list each name together with a description. Like Solarlog, Traccar, or La Marzocco do, for example. It does not have to be a table, could be just a list.

Reply:
Yea, would you mind opening an issue for that, as you've already mentioned, has nothing to do with this PR?

Reply:
true. 
For this PR, maybe you could already start the new structure by moving the switch description into a separate Switch section.

Reply:
Honestly, I'd like to just do that in the next PR to keep things consistent. I mean, right here, it's just one sentence

Reply:
Let's do the bigger changes in a seperate PR 😉 

./Klaas